src/os/ui/featureedit.js

```diff
@@ -689,6 +689,8 @@ os.ui.FeatureEditCtrl.prototype.disposeInternal = function() {
     this.previewFeature = null;
   }
 
+  goog.dispose(this.interaction);
```

Comment:
This should also remove the interaction from the map if defined. OL doesn't have any dispose logic of its own, so we could either remove the interaction here (where the logic to add it exists) or in the dispose function for our override (to ensure it's removed from the map no matter where it's being used).

I would probably lean toward putting it in the dispose function if `this.getMap()` is defined on the interaction so this case is handled everywhere the interaction is used.

Reply:
Yeah it's now only used in these 2 places (`modifyarea` and `featureedit` directives) so all of the dispose logic should be able to remove itself from the map.